testing/git-secret/APKBUILD

```
@@ -4,7 +4,7 @@
 # Note: git-secret is very badly designed, I'll replace this abuild once
 #   I find or write something better.
 pkgname=git-secret
-pkgver=0.2.2
+pkgver=0.2.5
 pkgrel=1
```

The pkgrel can be reset to zero because you have moved to a new pkgver

```suggestion
pkgrel=0
```

---

Good to know, thanks for pointing out! I'm coming from Arch Linux, where all packages starts with `pkgrel=1`.